tests/envs/test_make.py

```diff
@@ -0,0 +1,180 @@
+"""Tests that gym.make works as expected."""
+
+import re
+
+import pytest
+
+import gym
+from gym.envs.classic_control import cartpole
+from gym.wrappers import AutoResetWrapper, OrderEnforcing, TimeLimit
+from gym.wrappers.env_checker import PassiveEnvChecker
+from tests.envs.register_during_make_env import RegisterDuringMakeEnv
+from tests.envs.utils import all_testing_env_specs
+from tests.wrappers.utils import has_wrapper
+
+
+class ArgumentEnv(gym.Env):
+    observation_space = gym.spaces.Box(low=0, high=1, shape=(1,))
+    action_space = gym.spaces.Box(low=0, high=1, shape=(1,))
+
+    def __init__(self, arg1, arg2, arg3):
+        self.arg1 = arg1
+        self.arg2 = arg2
+        self.arg3 = arg3
+
+
+gym.register(
+    id="test.ArgumentEnv-v0",
```

Comment:
I've never seen this kind of id. Wouldn't we want something like "test/ArgumentEnv-v0"? Probably not in the scope of this PR though.

Reply:
That is a good point, @RedTachyon what are your thoughts?